print/templates/index.html

```diff
@@ -31,6 +31,9 @@
 {# operator detailed view #}
     {% include 'operator_detailedview.html' %}
 
+{# pattern only view #}
+    {% include 'pattern_only.html' %}
+
```

Comment:
Only include if this should actually show:

`<div class="page pattern-onlyview" style="display: {{ 'block' if view.pattern_onlyview else 'none' }}">{% include 'pattern_only.html' %}</div>`